src/GeneratedHydrator/ClassGenerator/HydratorGenerator.php

```diff
@@ -4,47 +4,18 @@
 
 namespace GeneratedHydrator\ClassGenerator;
 
-use CodeGenerationUtils\Visitor\ClassImplementorVisitor;
-use GeneratedHydrator\CodeGenerator\Visitor\HydratorMethodsVisitor;
-use PhpParser\NodeTraverser;
-use PhpParser\Node\Name;
-use PhpParser\Node\Stmt\Class_;
-use PhpParser\Node\Stmt\Namespace_;
+use PhpParser\Node;
 use ReflectionClass;
-use Zend\Hydrator\HydratorInterface;
 
 /**
- * Generator for highly performing {@see \Zend\Hydrator\HydratorInterface}
- * for objects
- *
- * {@inheritDoc}
- *
- * @author Marco Pivetta <ocramius@gmail.com>
- * @author Pierre Rineau <pierre.rineau@makina-corpus.com>
- * @license MIT
+ * Interface for the hydrator generator
  */
-class HydratorGenerator implements HydratorGeneratorInterface
+interface HydratorGenerator
```

Comment:
Hmm, this is a major BC break: would have preferred an exclusion rule, but let's roll with it